packages/app-store/_utils/useAddAppMutation.ts

```diff
@@ -1,5 +1,6 @@
 import type { UseMutationOptions } from "@tanstack/react-query";
 import { useMutation } from "@tanstack/react-query";
+import { useRouter } from "next/router";
```

Comment:
next/router is deprecated. We should be using next/navigation instead. This is causing all future routes to break.

Reply:
@joeauyeung more info on this https://nextjs.org/docs/pages/building-your-application/upgrading/app-router-migration#step-5-migrating-routing-hooks